.github/workflows/reuse.linux.yml

```diff
@@ -189,6 +189,35 @@ jobs:
         with:
           name: test-sqllogic-standalone-${{ matrix.handlers }}-${{ matrix.dirs }}
 
+  sqllogic_standalone_udf_server:
```

Comment:
Ping @everpcpc to check the ci.

Reply:
CI failed:
https://github.com/datafuselabs/databend/actions/runs/6098657496/job/16549342052?pr=12729

Reply:
fixed